internal/serve/graphql/resolvers/mutations.resolvers.go

```diff
@@ -237,8 +237,9 @@ func (r *mutationResolver) CreateFeeBumpTransaction(ctx context.Context, input g
 		}
 	}
 
-	tx, ok := genericTx.Transaction()
-	if !ok {
+	// Transaction() returns (tx, false) when genericTx is a fee-bump transaction
+	tx, isGenericTx := genericTx.Transaction()
+	if !isGenericTx {
```

Comment:
I don't think this makes sense. A "generic" transaction can be either a fee-bump or normal transaction, so there is no such thing as a transaction that isn't generic.

Can we just add a comment above the existing code to note that if `.Transaction()` doesn't return `ok`, then that means its a fee-bump?